Vimeo-Native/VimeoInterop/VimeoInterop.vcxproj.filters

```diff
@@ -24,6 +24,9 @@
     <ClInclude Include="resource.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="..\Include\Vimeo\VimeoAPI.h">
```

Comment:
This might be something that someone else should look at, but it looks weird that we're adding a header to this file when we're just upgrading to VS2015.